corgi/tasks/rhel_compose.py

```diff
@@ -33,65 +28,14 @@ def save_compose(stream_name) -> None:
             compose_url, variants
         )
         for key in "srpms", "rhel_modules":
-            no_of_relations += _create_relations(compose_data, key, compose_id, stream_name)
+            if key not in compose_data:
+                continue
```

Comment:
Should we raise an exception or log an error in case some data we expect is missing? When does this happen and why?

Reply:
It's expected in some cases that "rhel_modules" key does not exist in compose_data dict returned by RhelCompose.fetch_compose_data, that's all this is checking for. Most composes don't have modules, in fact I think only the appstream compose in RHEL 8 has modules. But that may change in the future.

Reply:
Maybe a comment here to explain that would be good, otherwise sounds good to me.